site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-siteInfo-list.vm

```diff
@@ -282,8 +289,17 @@
 							#end
 							</ul>
 						#else
-							#if ($!siteJoinable)
-								$tlang.getString("sitegen.siteinfolist.sitecan")
+							#if ($!siteJoinable && $joinLimitedAccountTypes.size() > 0)
+								$tlang.getFormattedMessage("sitegen.siteinfolist.sitecan", $uiService)
+								<ul>
+									#foreach( $accountType in $joinLimitedAccountTypes )
+										#foreach( $accType in $joinableAccountTypes )
+											#if( $accountType == $accType.type )
+                                                <li>$accType.label</li>
```

Comment:
The indentation here looks wrong, could you try to match it with the surrounding context, the issue maybe because of differences tabs and spaces.